.github/workflows/test-fuzz.yml

```diff
@@ -0,0 +1,35 @@
+# name: Forge Fuzz Tests
```

Comment:
Cause we don't have any fuzz tests in this template.
But it can be activated by uncommenting it once fuzz tests are added to any project that will use this template.